### DISCUSSION TITLE
Rectangle Episode 7: Redis, Angular+Wiz, Solid Start

### DISCUSSION THREAD

**Discussion by nmn** [rectanglepodcast-com/rectanglepodcast.com#62] [Ideas]
Redis, 

Angular+Wiz,

Solid Start 1.0 RC,

**Comment by PatrickJS** [comment]
some topics on redis https://github.com/rectanglepodcast-com/rectanglepodcast.com/discussions/41


* why we need in-memory database?
* redis was the first few during the new NoSQL era
* open-source lessons from Node.js and Joyent
* Express.js (sold by TJ)
